src/core/Akka/IO/Tcp.cs

```diff
@@ -762,7 +762,7 @@ internal MessageDispatcher FileIoDispatcher
         }
     }
 
-    public class TcpMessage
+    /*TODO: this class is not used*/public class TcpMessage
```

Comment:
These are factory methods which can be used by a Akka IO client. In JVM Akka this is for the Java API. I generally just used the message constructor:
new Tcp.Connect(...)  vs
TcpMessage.Connect(..)

I suppose we can either remove the TcpMessage factory class or keep it and make the message constructors internal.